app/uk/gov/hmrc/serviceconfigs/service/ServiceRelationshipService.scala

```diff
@@ -46,8 +47,9 @@ class ServiceRelationshipService @Inject()(
   def updateServiceRelationships(): Future[Unit] =
     for {
       slugInfos       <- slugInfoRepository.getAllLatestSlugInfos
+      knownServices   =  slugInfos.map(_.name)
       (srs, failures) <- slugInfos.toList.foldMapM[Future, (List[ServiceRelationship], List[String])]( slugInfo =>
-                           serviceRelationshipsFromSlugInfo(slugInfo)
+                           serviceRelationshipsFromSlugInfo(slugInfo, knownServices)
```

Comment:
you could just do `serviceRelationshipsFromSlugInfo(slugInfo, knownServices = slugInfos.map(_name))`  since knownServices is only used once